crates/uv/tests/it/snapshots/it__ecosystem__black-lock-file.snap

```diff
@@ -179,7 +179,6 @@ wheels = [
 
 [[package]]
 name = "black"
-version = "24.8.0"
```

Comment:
This is correct: the `pyproject.toml` we have checked-in here uses dynamic versioning.